### DISCUSSION TITLE
Denon x3700h and hoobs/homebridge

### DISCUSSION THREAD

**Discussion by Thebeatman** [community/community#134966] [Programming Help]
### Select Topic Area

Question

### Body

Anyone know if any of the plugins for hoobs such as denon tv or denon HEOS work with the x3700h @ grzegorz914 I know you wrote one.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Thebeatman ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
